docs/hpx.qbk

```diff
@@ -90,7 +90,8 @@
 [def __boost_atomic__           [@http://www.boost.org/doc/libs/release/doc/html/atomic.html Boost.Atomic]]
 [def __boost_serialization__    [@http://www.boost.org/doc/libs/release/libs/serialization/doc/index.html Boost.Serialization]]
 
-[def __hwloc_downloads__        [@http://www.open-mpi.org/projects/hwloc/]]
+[def __hwloc_downloads__        [@http://www.open-mpi.org/software/hwloc/v1.11 Hwloc Downloads]]
+[def __hwloc_doc__              [@http://www.open-mpi.org/projects/hwloc/doc/v1.11.9 Hwloc Documentation]]
```

Comment:
Should we link to https://www.open-mpi.org/projects/hwloc/doc/ instead (no version)?

Reply:
Good point, less updating for us. What about the download link? There doesn't seem to a page with all the downloads like for the docs. I think linking to the 1.11 download page is nicer in this case.

Reply:
Indeed.